data/sql/updates/pending_db_world/rev_1671313957796149100.sql

```diff
@@ -0,0 +1,199 @@
+SET @GUID := 25068;
+
+-- Raging Fire-Soul (Mangos)
+DELETE FROM `creature` WHERE `id1` IN (22311,22311,22311,22311,22311,22311,22309,22309,22310,22310,22310,22310,22310,22310,18127,19402,19402,19402,18867) AND `guid` BETWEEN @GUID+0 AND @GUID+18;
```

Comment:
duplicate deletes ? That does not seem to be needed

Reply:
Fixed 👍

Reply:
sorry for nitpicking :P